app/lib/dom.ts

```diff
@@ -0,0 +1,33 @@
+export function isInSubtreeOf(element: Element, target: Element) {
+  let current: Element | null = target;
+  while (current) {
+    if (current === element) return true;
+    current = current.parentElement;
+  }
+  return false;
+}
+
+export function setClassEnabled(element: HTMLElement, className: string, enabled: boolean) {
```

Comment:
i think this can be replaced with element.classList.toggle("class-name", false/true);

Reply:
TIL! Removed